src/components/Footer/FooterCfGov.tsx

```diff
@@ -1,24 +1,29 @@
 import { Icon } from '../Icon/Icon';
 import Footer from './Footer';
 
-export const FooterCfGov = (): JSX.Element => {
+export const FooterCfGov = ({
+  cfLink = 'https://www.consumerfinance.gov/'
```

Comment:
Removing trailing slash.  All of the links have double `//`.  They still work, the target page loads, but would still like it be formatted correctly. 
```suggestion
  cfLink = 'https://www.consumerfinance.gov'
```